plugin.json

```diff
@@ -1,13 +1,14 @@
 {
-    "id": "gke",
-    "version": "0.0.1",
+    "id": "gke-clusters",
```

Comment:
breaks existing installs, since it becomes another plugin. Do we want to rename? (maybe yes, before it's deployed widely)

Reply:
Let's do it. It will not really break, because since it's another plugin, it just means you'll have both side by side.